app/models/katello/host_available_module_stream.rb

```diff
@@ -37,6 +37,13 @@ def upgradable?
                                              :stream => available_module_stream.stream}).exists?
     end
 
+    def install_status?
+      return 'Not installed' if installed_profiles.empty?
+      return 'Installed' if installed_profiles && status == 'disabled'
+      return 'Up-to-date' if installed_profiles && status == 'enabled' && upgradable? == false
+      return 'Upgradable' if installed_profiles && status == 'enabled' && upgradable?
+    end
```

Comment:
The `return` keyword isn't very idiomatic in Ruby (except for early returns). I think this `case` statement should be the same:
```suggestion
    def install_status
      return 'Not installed' if installed_profiles.blank?
      case status
      when 'disabled'
        'Installed'
      when 'enabled'
        upgradable? ? 'Upgradable' : 'Up-to-date'
      end
    end
```

also note that I removed the question mark from `install_status` because question marks should only be used if the method returns `true` or `false`.